src/script/components/Modals/PrimaryModal/PrimaryModal.tsx

```diff
@@ -439,14 +449,17 @@ export const PrimaryModalComponent: FC = () => {
                     placeholder={inputPlaceholder}
                     required
                     data-uie-name="backup-password"
-                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-                      updatePasswordWithRules(event.target.value)
-                    }
+                    markInvalid={isBackupPasswordInvalid}
```

Comment:
And there You can check: 
```suggestion
                    markInvalid={isFormSubmitted && isBackupPasswordInvalid}
```